### PR TITLE
Update sponsor blog posts

### PR DESCRIPTION
**Pull request by miketraverso**

### CODE REVIEW

**Comment by Splaktar** [comment]
I'll take a look later tonight. I just got Juniper's input for a Vaadin blog post, so I should be able to add that in the process as well.

File: app/data/sessions.json [hunk 1]
```diff
@@ -1,30 +1,35 @@
 {
   "1001": {
     "id": 1001,
+    "track": "General",
```
Comment (Splaktar) [inline app/data/sessions.json]: What do these `track` fields do? The tracks are defined in `schedule.json`.

File: app/data/blog.json [hunk 2]
```diff
@@ -70,5 +70,14 @@
     "secondaryColor": "#3F51B5",
     "image": "/images/posts/speaker_announce.jpg",
     "brief": "The full speaker lineup for DevFest Florida has been announced. We're thrilled to have such a knowledgeable and entertaining speaker lineup for our inaugural DevFest Florida event. Check them out and get your tickets now!"
+  },
+  {
+    "id": "capgemini",
+    "title": "Capgemini - DevFest Florida Gold Sponsor",
+    "posted": "2016-10-28",
+    "primaryColor": "#ffdf11",
+    "secondaryColor": "#FFffdf11",
```
Comment (Splaktar) [inline app/data/blog.json]: Why is this 8 chars long?

**Review by Splaktar — COMMENTED** [review]
Working on adding the start of the Vaadin blog post here as well.

**Comment by miketraverso** [comment]
Added Tracks for the app I'm working on. Please leave this in. 

8 char HTML includes the alpha color. I was playing with that earlier.

**Comment by Splaktar** [comment]
Ah, the HEX w/ RGBA is only supported in the latest Chrome (52) and Firefox (49) browsers. So it won't work on Edge or IE11 (not supported atm by our site). I'm messing with the colors a little bit anyhow.

Any chance that you can put together one of those nicely formatted sponsor images for Vaadin?

**Comment by Splaktar** [comment]
screen shot 2016-10-29 at 12 27 28 am

On the actual blog page, the white backgrounds cause the blog title, navigation tabs, conf date and logo to be hidden. We probably need some amount of color, even if light in these background images.

**Comment by miketraverso** [comment]
Yep. First thing in the morning. That work?

**Comment by Splaktar** [comment]
Yep, no problem. I fixed the web URL links. Added Vaadin blog post and links. Added Nativescript links. Deploying and pushing for now then heading to bed.

File: app/data/sessions.json (hunk 1 above)
Comment (Splaktar) [reply app/data/sessions.json]: Turns out that this has broken all of the track listings on the schedule page. :(

**Comment by Splaktar** [comment]
Before:
screen shot 2016-10-29 at 1 08 17 pm

After (notice room locations missing):
screen shot 2016-10-29 at 1 08 03 pm

**Review by Splaktar — APPROVED** [review]

**Pull request merged**